### PR TITLE
Set tooltip to an empty string after removing texture/material

### LINKED ISSUE #32570: Inspector Dock is not clearing tooltip upon deletion
**Godot version:**
master

**OS/device including version:**
Windows 10

**Issue description:**
When hovering over empty Material slot (whose previously assigned to), the tooltip will still display a text. Hovering other slots which were never assigned anything shows the correct behavior of ... not showing anything.

**Steps to reproduce:**
1. Fill the Material slot with something
2. Clear it
3. Hover mouse over it.

not_clearing_cache

**Minimal reproduction project:**
Any project with a Material, I guess. I tried this with Godot's Material demo.

### PR DESCRIPTION
**Pull request by jmorton06**
Fixes #32570

### CODE REVIEW

File: editor/editor_properties.cpp [hunk 1]
```diff
@@ -2649,6 +2649,7 @@ void EditorPropertyResource::update_property() {
 	if (res == RES()) {
 		assign->set_icon(Ref<Texture>());
 		assign->set_text(TTR("[empty]"));
+		assign->set_tooltip(TTR(""));
```
Comment (akien-mga) [inline editor/editor_properties.cpp]: No need for `TTR()`, that's a macro for translatable strings, and `""` doesn't need translation :)

Comment (akien-mga) [reply editor/editor_properties.cpp]: Could you also squash commits together?

Comment (jmorton06) [reply editor/editor_properties.cpp]: Ah okay thanks :). Yeah sure. Will that require a new pull request?

Comment (akien-mga) [reply editor/editor_properties.cpp]: No, you can squash the commits in your local branch, and force push the changes to your remote to update the pull request.
See instructions on http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

Comment (jmorton06) [reply editor/editor_properties.cpp]: Thank you. That should be done now

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**